site/content/docs/monitoring/check-pricing.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Check Pricing
+weight: 9
+menu:
+   platform:
+    parent: "Monitoring"
+---
+
+Check pricing is based on a number of factors. Here is a breakdown of the things that determine the cost of a check.
```

Comment:
@drakirnosslin would a breakdown summary of check types at the beginning here help people who might be new to Checkly quickly understand what is and is not relevant to their case?

Reply:
no, let's keep the page focused. It needs to be short and scannable.

Reply:
@tnolet you're probably right. Would you structure it like this?
<img width="826" alt="Screenshot 2024-01-03 at 15 12 29" src="https://github.com/checkly/docs.checklyhq.com/assets/12794903/69feaa78-f000-4784-8b40-0dfccfd836fe">